Cargo.toml

```diff
@@ -51,6 +45,8 @@ bytesize = "1.3.0"
 chrono = { version = "0.4.38", default-features = false, features = [
   "clock",
 ] }
+ratatui = "0.26.1"
```

Comment:
is it used now ?

Reply:
no, it can be removed now, but it will be introduced in the future

Reply:
could you please remove it?

we could also enable udep like:
https://github.com/uutils/coreutils/blob/main/.github/workflows/CICD.yml#L78-L88

Reply:
> could you please remove it?
> 
> we could also enable udep like: https://github.com/uutils/coreutils/blob/main/.github/workflows/CICD.yml#L78-L88

I made a pr for this #57